inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/OpenStreamTransformController.java

```diff
@@ -89,4 +98,11 @@ public Response<Boolean> delete(@Validated DeleteTransformRequest request) {
         Preconditions.expectNotNull(LoginUserUtils.getLoginUser(), ErrorCodeEnum.LOGIN_USER_EMPTY);
         return Response.success(streamTransformService.delete(request, LoginUserUtils.getLoginUser().getName()));
     }
+
+    @RequestMapping(value = "/transform/function/list", method = RequestMethod.POST)
+    @ApiOperation(value = "Get transform function docs list with optional type filtering and pagination")
```

Comment:
Also add to `StreamTransformController`.

Reply:
done